utils/wasm-builder/src/builder.rs

```diff
@@ -93,6 +95,8 @@ pub struct WasmBuilder {
 	project_cargo_toml: PathBuf,
 	/// Features that should be enabled when building the wasm binary.
 	features_to_enable: Vec<String>,
+	/// Should the builder check that the `runtime_version` section exists in the wasm binary?
+	check_for_runtime_version_section: bool,
```

Comment:
Nit. 
Since the default is now to check for runtime version and the default for a bool is `false`.
Maybe we can invert the logic here. That is, have this flag named `disable_runtime_version_check`.
Default would be false and we have a `disable_runtime_version_section_check` function setting this flag to true (the non default value)